docs/pages/versions/unversioned/sdk/filesystem.md

```diff
@@ -323,6 +323,40 @@ Returns a Promise that resolves to an object with the following fields:
 
 - **uri (_string_)** -- A `content://` URI pointing to the file. This is the same as the `fileUri` input parameter but in different format.
 
+### `FileSystem.getFreeDiskStorageAsync()`
```

Comment:
@ninjaachibi @vivianzzhu91 it looks like the error handling behavior in both of these functions differs across platforms -- on Android if an exception is caught, the promise is rejected, whereas on iOS the promise is never rejected but might resolve with nil (undefined). Could you standardize this behavior across platforms and document whichever option you choose? Which one you choose should probably depend on whether you expect the exception/null case to happen in any reasonable scenarios -- if so it might good for the user to expect to handle an occasional `null`/`undefined` value. Otherwise, rejecting the promise would signify that something extra bizarre has happened.